Reference/Searching/Examine/quick-start.md

```diff
@@ -1,30 +1,31 @@
 # Quick start
 
-**Applies to: Umbraco 4.11.5+**
-
-_This guide will help you get setup quickly using Examine with minimal configuration options. Umbraco ships Examine with 2 internal indexes which should not be used for searching when returning results on a public website because it indexes content that has not been published yet. So to get started we need to create a new index._
+**Applies to: Umbraco 7+**
 
+_This guide will help you get setup quickly using Examine with minimal configuration options. Umbraco ships Examine with 2 internal indexes which should not be used for searching when returning results on a public website because it indexes content that has not been published yet. It also ships with an external index that you can use to get up and running.._
 
 ## Create an index
 
-To create a searchable index we need to create 3 things: an **Indexer** a **Searcher** and an **Index Set**.
+To create a searchable index we need to create 3 components: an **Indexer** a **Searcher** and an **Index Set**. Umbraco 7 ships with these three components ready to go, but you can use the below as a reference as you walk through your implementation.
 
-1. Open ~/Config/ExamineSettings.config and add an *indexer* under the 'ExamineIndexProviders/providers' section (in this example it is named *ExternalIndexer*):
+1. Open ~/config/ExamineSettings.config and add an *indexer* under the 'ExamineIndexProviders/providers' section (in this example it is named *ExternalIndexer*):
```

Comment:
The folder name is with upper case in an umbraco install.  Why would we lowercase this?

Reply:
Hmm, good point! Must be a VS issue for me then - downloading the .zip keeps it as uppercase but some of my older projects had the lowercase. I'll revert this one!